### PR TITLE
Aqa/js objects

### PR DESCRIPTION
**Pull request by manatskaya**
Hi Luibov,
Please review when you feel better and have time. 
Thank you
Best regards,
Svitlana

### CODE REVIEW

**Review by landriiuk — APPROVED** [review]

File: js-objects/objects.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+   if (val === "The Great Gatsby") {
+        continue;  
+    }
+   if (val > "1940" && typeof val === "number") {
+    console.log("> 1940", val);
+   }
+    if (val === "George Orwell")
+      book[prop] = "New author";
+}
+}
```
Comment (landriiuk) [inline js-objects/objects.js]: Add formatting here. 
```suggestion
    } 
}
```

File: js-objects/objects.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    },
+    user_2: {
+        languageName: "Ukrainian",
+        languageLevel: "Native"
+        },
+    user_3: { 
+        languageName:"German",
+        languageLevel: "A1",
+}
+}
```
Comment (landriiuk) [inline js-objects/objects.js]: and here

File: js-objects/objects.js [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+    age: 25,
+    email: "anna@london.com",
+    address: 
+    {
+    city: "London",
+    postalCode: "E1 7DS",
+    } 
+}
+
+
```
Comment (landriiuk) [inline js-objects/objects.js]:
```suggestion

```

Comment (landriiuk) [reply js-objects/objects.js]: Delete unnecessary row

**Comment by landriiuk** [comment]
Your task is good, thank you. 
Pls, ensure that you have 1 empty row between some functions, also don't forget about formatting.

**Comment by landriiuk** [comment]
Check comments, and merge this pr

**Comment by manatskaya** [comment]
Hi Luibov,
Thank you for your feedback. Will take your comments into account. Updated accordingly.
Svitlana

**Pull request merged**